tests/tasks/generators/clusterloader/load.yaml

```diff
@@ -67,11 +70,36 @@ spec:
       # we are not testing PVS at this point
       CL2_ENABLE_PVS: false
       PROMETHEUS_SCRAPE_KUBE_PROXY: true
+      PROMETHEUS_SCRAPE_APISERVER_ONLY: true
       ENABLE_SYSTEM_POD_METRICS: false
       NODE_MODE: master 
       EOL
       cat $(workspaces.source.path)/overrides.yaml
       cp $(workspaces.source.path)/overrides.yaml $(workspaces.results.path)/overrides.yaml
+
+      # TODO: Currently pathing the prometheus manifests for remote write. Move this to upsteam going forward.
+      echo "volumeBindingMode: WaitForFirstConsumer" >> $(workspaces.source.path)/perf-tests/clusterloader2/pkg/prometheus/manifests/0ssd-storage-class.yaml
+      cat $(workspaces.source.path)/perf-tests/clusterloader2/pkg/prometheus/manifests/0ssd-storage-class.yaml
+      cat << EOF >> $(workspaces.source.path)/perf-tests/clusterloader2/pkg/prometheus/manifests/prometheus-prometheus.yaml
+        containers:
+          - name: aws-sigv4-proxy-sidecar
```

Comment:
Sidecar works. But have you considered using the built-in support of SigV4 which is available since [2.26.0](https://aws.amazon.com/blogs/opensource/prometheus-2-26-0-adds-aws-signature-version-4-support/).
Ref: https://github.com/prometheus-operator/prometheus-operator/blob/main/pkg/apis/monitoring/v1/types.go#L1074-L1088
IMO using the prometheus operator's API may make it easier to upstream the change.

Reply:
This comment doesn't block this PR from merging as long as we still use AMP and AMG.
I'm OK to address this in a followup PR if needed.

Reply:
yeah thats the plan. i tried upgrading the prometheus and prometheus operator, but with the new versions available in the gcr repository the prometheus pods fails to come up with error in parsing configmap. I will try to raise a fix to this upstream and we can remove this patch going forward, but we can keep this atleast till 1.26.

Reply:
What prometheus version does cluster-loader currently use?

Reply:
v2.25 https://github.com/kubernetes/perf-tests/blob/release-1.23/clusterloader2/pkg/prometheus/manifests/prometheus-prometheus.yaml#L50

Reply:
can you add a backlog issue for tracking this @ashishranjan738 and assign a label as 1.26.

Reply:
yeah created here https://github.com/awslabs/kubernetes-iteration-toolkit/issues/326

Reply:
cool, thanks. I added label 1.26 to it.